### PR TITLE
Add `init` command

### LINKED ISSUE #67: CLI: Generate template config
`virtualship init`: create template yaml file and expedition directory

### PR DESCRIPTION
**Pull request by VeckoTheGecko**
Fixes #67

Changes:
- Add `virtualship init` command
- link to oceanparcels.org in docs sidebar
- add README badges and link to repo
- Fixes:
  - fix documentation config
  - remove `meta.yml` (managed by the feedstock)

---
## Good to know

`virtualship` now comes with a `virtualship.static` folder which is used to store static files (e.g., example configuration `.yml` files). For data files (which are larger, and aren't as core to the functionality of virtualship) shouldn't be put here but instead should be downloaded on demand.

`virtualship.utils` contains a bunch of utilities for `virtualship`, including convenience functions for reading these static files

### CODE REVIEW

File: src/virtualship/cli/commands.py [hunk 1]
```diff
@@ -15,7 +17,26 @@
...
-    raise NotImplementedError("Not implemented yet.")
+    path = Path(path)
+    path.mkdir(exist_ok=True)
+
+    config = path / SHIP_CONFIG
+    schedule = path / SCHEDULE
+
+    if config.exists():
+        raise FileExistsError(
+            f"File '{config}' already exist. Please remove it or choose another directory."
```
Comment (erikvansebille) [inline src/virtualship/cli/commands.py:28]: "directory" or "path"?

Comment (VeckoTheGecko) [reply src/virtualship/cli/commands.py:28]: Directory. The `path` varname is the path to the directory

**Review by erikvansebille — APPROVED** [review]

**Comment by VeckoTheGecko** [comment]
Static files aren't being packaged properly despite working locally. Removing the `--no-build-isolation` flag would fix it, but I'm not 100% sure if that's the right fix. Waiting on reponse in https://github.com/pypa/setuptools/discussions/4694

**Comment by VeckoTheGecko** [comment]
Decided to just remove the flag `--no-build-isolation`

**Pull request merged**